BrainPortal/app/models/data_usage.rb

```diff
@@ -0,0 +1,77 @@
+
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2022
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# Model representing usage of files in select groups
+class DataUsage < ApplicationRecord
+
+  Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
+
+  validates_presence_of :user_id
+  validates_presence_of :group_id
+  validates_presence_of :yearmonth
+
+  belongs_to :user,  :optional => false
+  belongs_to :group, :optional => false
+
+  validates_uniqueness_of :user_id, :scope => [ :group_id, :yearmonth ]
+
+  def self.increase_views(user, userfile)
```

Comment:
Should have :nodoc: or documentation no?
Same comment for other methods here.

Reply:
Yes, will make the adjustments.